fabric-networking-blockentity-v0/src/main/java/net/fabricmc/fabric/api/block/entity/BlockEntityClientSerializable.java

```diff
@@ -26,4 +29,19 @@
 	void fromClientTag(CompoundTag tag);
 
 	CompoundTag toClientTag(CompoundTag tag);
+	
+	/**
+	 * Call this method on the server to schedule a BlockEntity sync to client. This will call
+	 * {@link #toClientTag(CompoundTag)} on the server to generate the packet data, and then
+	 * {@link #fromClientTag(CompoundTag)} on the client to decode that data.
+	 */
+	default void sync() {
+		World world = getWorld();
+		if (world instanceof ServerWorld) {
+			((ServerWorld)world).method_14178().markForUpdate(getPos());
+		}
+	}
+
+	public World getWorld();
+	public BlockPos getPos();
```

Comment:
Unnecessary `public` here (the other methods in the interface don't have them either).